app/move/steps/create.js

```diff
@@ -47,6 +48,17 @@ const riskStep = {
   ],
 }
 
+function isYouthMove(fieldValue) {
```

Comment:
I'm aware this is introducing some untested methods. However, the addition of these is temporary to support the ability to feature flag this change.

Once this feature flag is removed it will remove all of these additions and allow this to be tidied up. I'm going to do the follow up PR now so that we can see the immediate changes coming.

Reply:
mmmmkay ;0